packages/console/src/containers/AppContent/components/UserInfo/index.tsx

```diff
@@ -39,7 +39,7 @@ const UserInfo = () => {
     update,
   } = useUserPreferences();
 
-  if (!user) {
+  if (isLoadingUser) {
```

Comment:
did we have alignment of using `isLoading` vs `!data`?

Reply:
Yes, I think we did. `isLoading` is equivalent to `!data && !error`, while `!data` doesn't care about if the request has ended up with some error, in which case we should not show the loading.